tests/dragonfly/replication_test.py

```diff
@@ -2320,7 +2323,7 @@ async def test_master_stalled_disconnect(df_factory: DflyInstanceFactory):
     c_master = master.client()
     c_replica = replica.client()
 
-    await c_master.execute_command("debug", "populate", "200000", "foo", "500")
+    await c_master.execute_command("debug", "populate", "200000", "foo", "500", "RAND")
```

Comment:
why did you change these tests?

Reply:
because we use compression now the debug populate is very compressed unless using the RAND option and therefore the test would fail as the master will send all the data to socket and wont get blocked. We need to get blocked on write to socket and we need to have enough data to send to be blocked